stanford-earth/matson
# EARTH-406: Section Highlight Cards.

# READY FOR REVIEW

# Summary
- Section Header Cards Styles

# Needed By (Date)
- Today if possible

# Urgency
- Urgent

# Steps to Test

1. Check out branch EARTH-406 in matson, stanford_components, stanford_paragraph_types
2. Enable stanford_paragraph_section_highlight_cards
3. Revert the feature and clear cache
4. Create a section highlight cards paragraph type on a complex page
5. Review style

# Associated Issues and/or People
-EARTH-406

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
